src/main/thrift/parquet.thrift

```diff
@@ -194,6 +194,13 @@ enum FieldRepetitionType {
 /**
  * Statistics per row group and per page
  * All fields are optional.
+ *
+ * For BinaryStatistics in Parquet, we want to distinguish between the statistics derived from
+ * comparisons of signed or unsigned bytes.
+ * By default, Parquet will compare Binary type data as a signed bytestring, and this is the
+ * default behavior for filter pushdown when signed and unsigned are not specified in the
+ * Statistics. However, when signed_min, signed_max, unsigned_min and unsigned_max are all
+ * specified, the client has the option of using either signed or unsigned bytestring statistics.
```

Comment:
How about we say for a given file, it's binary stats will be written 1 of 3 ways:

1) All min / max fields are not populated -- no stats available
2) only fields 1 and 2 are populated , but not fields 5,6,7,8 -- only signed min/max available
3) only fields 5,6,7,8 are populated -- both signed / unsigned stats available, fields 1 + 2 are deprecated for binary fields from now on